galley/docker/testing/init.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
Looks like this follows the simple recipe at https://docs.docker.com/config/containers/multi-service_container. Did you look into supervisord at all for multi-process container?